server/rest/phase.py

```diff
@@ -27,6 +28,19 @@
 from girder.utility.progress import ProgressContext
 
 
+def _loadMetadata(params):
+    if 'meta' not in params:
+        return
```

Comment:
Since the calling code uses the return value of this function, I think it would be clearer to return `None` in this case.